### DISCUSSION TITLE
Oversizing Skins

### DISCUSSION THREAD

**Discussion by Uggers** [SFX-WoW/Masque#192] [Questions] [answered]
I use the dream skin and since the latest update Masque is making it bigger than usual.

Version 8.1.0-Alpha works fine with the correct size.
but none of the other seems to do it. I don't know if dream skin has an issue or masque.

**Comment by StormFX** [comment]
The default skins were adjusted to be more inline with proper button sizes. Dream was only increased by 6 pixels (3 on each side). If they're overlapping or otherwise too big for your tastes, use the add-on's settings to adjust the button size and/or spacing.

**Comment by Uggers** [comment]
Ahh sorry I didn't realise the change,

Thanks i'll update and tweak to the size i prefer.

Thanks again

**Comment by StormFX** [comment]
You're welcome.

**Comment by Uggers** [comment]
Sorry to be a pain but were do I find the options.

Skin settings has very little options for me at all;
[image: image.png]

**Comment by StormFX** [comment]
You need to change the size in the add-on that's being skinned by Masque. Eg, your action bar, buff or bag mod. Masque doesn't support size changing.

**Comment by Uggers** [comment]
Hey,

So I've tried multiple times to get bartender to adjust to the old dream
method but its not possible as it doesn't allow the adjustments without
doing each individual bar and the spacing isn't as exact.
Is there a way to make a "old-dream" skin at all?

I tried to a few times but I just can't get it to work or even adjust
modifying your dream file? I appreciate you don't want to maintain another
version but I'm unsure how to fix it back to how I used to have it.
Any help would be greatly appreciated

Kind regards

**Comment by StormFX** [comment, marked as answer]
As I said, I only adjusted the size of Dream by a few pixels (2 on each side). I can't be bothered going through all the changes I made that affect whatever is you're trying to accomplish to create an alternate version for one user.

Bartender allows you to adjust the scale and spacing of the buttons. If you're unable to get it how you want it with Bartender, try a different Action Bar add-on, like Dominos. If that doesn't work, you can copy the `Masque\Skins\Dream.lua` file to `Masque\Skins\Dream2.lua`, open `Masque\Skins\Skins.xml` and change:

```xml
	<Script file="Dream.lua"/>
```
to
```xml
	<Script file="Dream.lua"/>
	<Script file="Dream2.lua"/>
```
Then, open your new `Dream2.lua` and change:

```lua
	Core.AddSkin("Dream", {
```
to

```lua
	Core.AddSkin("Dream2", {
```
Once that's done, you can go into `Dream2.lua` and edit all of the `Width` and `Height` values to your liking. Or you can just edit the original `Dream.lua`. In eithe case, make sure you back it up.

**Comment by Uggers** [comment]
Perfect thank you

**Comment by StormFX** [comment]
You should be able to subtract 4 from each value and get the desired result.

**Comment by StormFX** [comment]
By the way, in case you didn't know: You can manually type numbers, including decimals, into Bartender's size and padding options. That would probably be easier than editing a skin. :p

**Comment by Uggers** [comment]
Ah doh! I was using the slider to adjust, that explains it then.

Appreciate you taking a look even though you didn't have to do that.

I'll try again with bartender first
